examples/python/geometry/point_cloud_paint.py

```diff
@@ -25,15 +25,11 @@
 # ----------------------------------------------------------------------------
 
 import open3d as o3d
-import os, sys
-
-pyexample_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-test_data_path = os.path.join(os.path.dirname(pyexample_path), 'test_data')
-sys.path.append(pyexample_path)
 
 if __name__ == "__main__":
     print("Load a ply point cloud, print it, and render it")
-    pcd = o3d.io.read_point_cloud(os.path.join(test_data_path, 'fragment.ply'))
+    sample_ply_data = o3d.data.DemoCropPointCloud()
+    pcd = o3d.io.read_point_cloud(sample_ply_data.pointcloud_path)
```

Comment:
please check if this is required, or we can manage with `SamplePointCloudPLY/PCD` ?

Reply:
Used SamplePointCloudPLY here